ethcore/src/engines/mod.rs

```diff
@@ -71,6 +71,8 @@ pub trait Engine : Sync + Send {
 	/// Block transformation functions, after the transactions.
 	fn on_close_block(&self, _block: &mut ExecutedBlock) {}
 
+	/// If true, generate_seal has to be implemented.
+	fn seals_internally(&self) -> bool { false }
```

Comment:
There is plenty of methods next to each other in trait definitions in this and other files in the codebase. Id argue that for definitions its fine, as it often groups method families.